CHANGELOG.md

```diff
@@ -296,7 +296,7 @@
 # Version 0.11.1 (2018-02-19)
 
 - Fixed windows not receiving mouse events when click-dragging the mouse outside the client area of a window, on Windows platforms.
-- Added method `os::android::EventsLoopExt:set_suspend_callback(Option<Box<Fn(bool) -> ()>>)` that allows glutin to register a callback when a suspend event happens
+- Added method `os::android::EventsLoopExt:set_suspend_callback(Option<Box<dyn Fn(bool) -> ()>>)` that allows glutin to register a callback when a suspend event happens
```

Comment:
We shouldn't retroactively modify the changelog.

Reply:
Done.